### ISSUE TITLE
No Patch Address Found for either hook for Instagram v43

### ISSUE THREAD

**Issue by hitrain** [karek314/hexbytescanner-hooks#3]
Extracted thin arm64, was unable to find address with either pattern.

**Comment by karek314** [comment]
This one
```
[
  {
    "pattern": "FFB5E00313AAE103??AA??????????0300",
    "patchBytes": "02",
    "patchDistance": "0x3"
  }
]
```

Should work, I can look at this but not sooner than week or two from now. You can try locate tls_process_server_certificate method manually and modify cmp instruction nearby to trick into trust untrusted cert.

**Comment by hitrain** [comment]
Yes, i try run this for framework and found address, but after resign still getting ssl error

**Comment by hitrain** [comment]
mov        x21, x0
cmp        w21, #0x1
should changed to ?

**Comment by hitrain** [comment]
thanks, i did it!

**Comment by karek314** [comment]
Do you mind updating the pattern? I bet 1 or 2 bytes changed. I would appreciate pull request.
